### PR TITLE
Update default

### PR DESCRIPTION
**Pull request by caribouW3**

### CODE REVIEW

**Review by plehegar — APPROVED** [review]

File: template/default [hunk 1]
```diff
@@ -1,18 +1,29 @@
...
+ {% endif %}
 
-Please take a few minutes to browse and bookmark this information:
-  https://w3c.github.io/Guide/participant/group.html?gid={{ group.id }}
+Welcome to the {{ group.name }}!
 
-If you have not already done so, link your GitHub account to your W3C account:
+This is a short introduction email to provide useful links about the group.
+You will find a lot of information about the {{ group.name }} linked from 
+  https://www.w3.org/groups/{{ shorttype }}/{{ group.shortname }}
```
Comment (vivienlacourba) [inline template/default]: Add a trailing slash to groups pages URIs.

```suggestion
  https://www.w3.org/groups/{{ shorttype }}/{{ group.shortname }}/
```

File: template/default [hunk 2]
```diff
@@ -1,18 +1,29 @@
...
+
+In particular, the following page tells you where the {{ group.name }} works, including repositories
+  https://www.w3.org/groups/{{ shorttype }}/{{ group.shortname }}/tools
```
Comment (vivienlacourba) [inline template/default]: Add trailing slash to groups pages URIs:

```suggestion
  https://www.w3.org/groups/{{ shorttype }}/{{ group.shortname }}/tools/
```

File: template/default [hunk 3]
```diff
@@ -1,18 +1,29 @@
...
+
+For meetings, check the group calendar at
+  https://auth.w3.org/?url=https://www.w3.org/groups/{{ shorttype }}/{{ group.shortname }}/calendar
+
+If you have not already done so, please link your GitHub account to your W3C account using
   https://www.w3.org/users/myprofile/connectedaccounts
```
Comment (vivienlacourba) [inline template/default]: Add a trailing slash to the this URI as well:

```suggestion
  https://www.w3.org/users/myprofile/connectedaccounts/
```

File: template/default [hunk 4]
```diff
@@ -1,18 +1,29 @@
...
-If you have not already done so, link your GitHub account to your W3C account:
+This is a short introduction email to provide useful links about the group.
+You will find a lot of information about the {{ group.name }} linked from 
+  https://www.w3.org/groups/{{ shorttype }}/{{ group.shortname }}
+
+In particular, the following page tells you where the {{ group.name }} works, including repositories
+  https://www.w3.org/groups/{{ shorttype }}/{{ group.shortname }}/tools
+
+For meetings, check the group calendar at
+  https://auth.w3.org/?url=https://www.w3.org/groups/{{ shorttype }}/{{ group.shortname }}/calendar
```
Comment (vivienlacourba) [inline template/default]: Add trailing slash to groups pages URIs:

```suggestion
  https://auth.w3.org/?url=https://www.w3.org/groups/{{ shorttype }}/{{ group.shortname }}/calendar/
```

Comment (vivienlacourba) [inline template/default]: @plehegar, @caribouW3, I'm not so sure on the benefit to link to an auth.w3.org URI, as on the calendar it should already be clear that people need to log in if they want to see further information such as the joining details. e.g. it is currently written:

> **Joining Instructions**
> 
> Instructions are restricted to W3C users with Member access. You need to [log in](https://auth.w3.org/?url=https%3A%2F%2Fwww.w3.org%2Fevents%2Fmeetings%2Ffc01a32d-085d-486a-bae0-2411ef3a2225%2F20231018T120000%2F) to see them.

and

> **Log in to subscribe to this event**

**Review by vivienlacourba — CHANGES_REQUESTED** [review]

File: template/default (hunk 4 above)
Comment (caribouW3) [reply template/default]: I made all those changes.

**Review by vivienlacourba — APPROVED** [review]

File: template/default (hunk 4 above)
Comment (plehegar) [reply template/default]: actually, it's not always clear that you need to authenticate to see the join info.
and our own system do so, eg.
  https://lists.w3.org/Archives/Public/public-trace-context/2023Oct/0002.html

**Pull request merged**